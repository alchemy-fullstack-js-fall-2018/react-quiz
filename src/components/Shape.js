import React from 'react';

export default function ColorSelector({ fontColor, backgroundColor, text }) {
  return (
    <div style={{ background: backgroundColor, color: fontColor }}>{text}</div>
  );
}
