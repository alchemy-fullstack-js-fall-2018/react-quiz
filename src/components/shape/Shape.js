import React, { Fragment } from 'react';

const Shape = ({ text, textChange, colorChange }) => {
  return (
    <Fragment>
      <div>
        <span value={text} onChange={textChange} onColorChange={colorChange}></span>
      </div>
    </Fragment>
  );
};

export default Shape;
