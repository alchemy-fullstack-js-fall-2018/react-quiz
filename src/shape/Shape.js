import TextSelector from '../textSelector/TextSelector'
