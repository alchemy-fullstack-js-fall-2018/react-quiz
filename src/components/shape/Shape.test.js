import React from 'react';
import { shallow } from 'enzyme';
import Shape from './Shape';

describe('shape', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Shape
        text={'asdf'}
        fontColor={'black'}
        backgroundColor={'white'}
      />
    );
  });

  it('renders a shape', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
