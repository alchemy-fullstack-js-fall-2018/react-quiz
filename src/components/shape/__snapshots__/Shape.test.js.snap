// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shape renders a shape 1`] = `
<Fragment>
  <div
    style={
      Object {
        "backgroundColor": "white",
        "color": "black",
      }
    }
  >
    asdf
  </div>
</Fragment>
`;
